media-source/mp4/test-v-no-keyframe-segment-manifest.json

```diff
@@ -0,0 +1,4 @@
+{
+    "url": "mp4/mp4-main-ogop-h264ogop.mp4",
```

Comment:
Inspecting the binary mp4 file using Chrome/Chromium, I see:
Chrome M64 (Release build): chrome://media-internals contains 10 debug warnings: "Prepared video sample is not conformant" -- this is logged when the bitstream converter detected a problem (specifically, this method returned false for those 10 frames: https://cs.chromium.org/chromium/src/media/formats/mp4/avc.cc?rcl=61cae6f129576601cb0dd964a6ce61a41e82abe0&l=180)

tip-of-tree DEBUG build with proprietary codecs:
Same 10 warnings in chrome://media-internals
debug logs indicate these occur at the end of each media segment, from: https://cs.chromium.org/chromium/src/media/formats/mp4/avc.cc?rcl=61cae6f129576601cb0dd964a6ce61a41e82abe0&l=227

Regardless, the media parses and plays.

Also, unlike the webm binary, this file has just one track (video), and exactly one keyframe, and multiple media segments -- several that begin with nonkeyframes. I believe this matches the scenario you're trying to test support for (where not all media segments begin all their tracks with keyframes).

You might consider making the first media segment appended *not* begin with a keyframe. Compliant implementations must accept and drop such nonkeyframes until the first keyframe. This would be yet a stronger test of compliance.

Reply:
(All 10 of the extra debug logs for those nonconformat samples indicate "Unexpected NALU type 8 in order_state 2").